docusaurus.config.js

```diff
@@ -134,20 +138,7 @@ module.exports = {
               to: "cli/",
               label: "CLI",
             },
-            {
-              to: "protocol/",
-              label: "Protocol",
-            },
-            {
-              to: "https://celo.academy/c/tutorials/4",
-              label: "Tutorials",
-              target: "_blank",
-            },
-            { to: "showcase", label: "DApps" },
-            {
-              to: "community/guidelines",
-              label: "Community",
-            },
+            { to: "https://www.celopg.eco/ecosystem", label: "dApps", target: "_blank" },
           ],
```

Comment:
please delete these pages as well @sodofi , so they can't be founde/ referenced

Reply:
I think they should be found/referenced just not from tab navigation